pyphare/pyphare/pharein/simulation.py

```diff
@@ -395,6 +395,10 @@ def check_restart_options(**kwargs):
         if "mode" not in restart_options:
             raise ValueError (f"Restart mode not set, valid modes are {valid_modes}")
 
+        if "dir" in restart_options:
+            while restart_options["dir"][-1] == "/": # samrai can't handle this
```

Comment:
maybe should be 

```
while len(restart_options["dir"]) > 1 restart_options["dir"][-1] == "/
```

?

Reply:
actually 

```
while "//" in restart_options["dir"]: restart_options["dir"] = restart_options["dir"].replace("//", "/")
```

Reply:
or maybe just raise and say it's invalid

Reply:
"aaa/bbb///".rstrip("/")

Reply:
IMO we should also do other checks such as permissions (https://www.tutorialspoint.com/How-to-check-the-permissions-of-a-directory-using-Python), other things?

Reply:
check there's some space left on the disk maybe

Reply:
probably not straightforward with the quotas